Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.7-alpine
+
+RUN apk add --no-cache build-base gcc bash cmake git
+
+RUN gem install bundler -v "~>1.0" && gem install bundler jekyll
+
+WORKDIR /site
+
+RUN gem install github-pages 
+
```

Comment:
For now, I'm generating the gem file here on the fly. I'm not sure, if it may be better to generate it by docs-ts. Because that would enable the person that does not care about docker, but has jekyll aready installed, to just navigate to the docs folder and spawn jekyll.